README.md

```diff
@@ -15,5 +34,11 @@ In order to get started you will want to do a couple of things.
 * clone this document repository
 * create a branch with your request ID \(issue number provided when you requested resources\)
 
+## References
+* OpenLab Web Site is hosted on [OpenLab](https://openlabtesting.org/)
+* Join our IRC Channel `#askopenlab` on freenode.net
+* The OpenLab CI github APP is hosted on [github APP](https://github.com/apps/theopenlab-ci)
```

Comment:
I think we should switch the order of `github APP` and `CI jobs' status` in the list, let newbie to get interesting content at first